ACadSharp/Objects/MultiLeaderAnnotContextClasses.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+
+using ACadSharp.Attributes;
+using ACadSharp.Tables;
+
+using CSMath;
+
+
+namespace ACadSharp.Objects
+{
+
+	/// <summary>
+	/// Nested classes in MultiLeaderAnnotContext
+	/// </summary>
+	public partial class MultiLeaderAnnotContext : CadObject
+	{
+		/// <summary>
+		/// Represents a leader root
+		/// 302	DXF: “LEADER“
+		/// </summary>
+		public class LeaderRoot : ICloneable
+		{
+			public LeaderRoot() { }
+
+			/// <summary>
+			/// B	290	Is content valid (ODA writes true)
```

Comment:
Same as for [MultiLeaderAnnotContext.cs](https://github.com/DomCR/ACadSharp/pull/169/files#diff-f431954f2bc08d2f6071c89702472b01e443aada7ee63b093f5d00d6a82995ea)